### PR TITLE
branch#46 (Add lecturerView)

### PR DESCRIPTION
**Pull request by MariusRim**
Added LecturerView. It is now accessible from facultyView and has a link to create a course

### CODE REVIEW

File: Controller/Lecturer/LecturerController.cs [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        {
+            Lecturer = lecturer;
+        }
+
+        public void LoadLecturerData()
+        {
+            LecturerView.Lecturer = Lecturer;
+            LecturerView.CourseList.Items.Clear();
+            LecturerView.ReviewList.Items.Clear();
+            List<Course> courses = Lecturer.Courses.ToList();
```
Comment (Gintasp) [inline Controller/Lecturer/LecturerController.cs]: null check ar `Lecturer.Courses != null`

File: Controller/Lecturer/LecturerController.cs [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            List<Review> reviews = Lecturer.Reviews.ToList();
```
Comment (Gintasp) [inline Controller/Lecturer/LecturerController.cs]: Same null check ir cia

**Comment by MariusRim** [comment]
Also added null check

File: Controller/Lecturer/LecturerController.cs [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        }
+
+        public void HandlePreviousFormClick()
+        {
+            LecturerView.Hide();
+            FacultyView = new FacultyView(new FacultyController(), Lecturer.Faculty, Lecturer.Faculty.University);
+            FacultyView.Show();
+        }
+
+        //      Not implemented in the form
```
Comment (Gintasp) [inline Controller/Lecturer/LecturerController.cs]: Nepaliekam tokiu useless komentaru, trinam lauk viska.

File: View/Individual/LecturerView.cs [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        {
+            LecturerController.HandlePreviousFormClick();
+        }
+
+        private void OnFormClose(object sender, FormClosedEventArgs e)
+        {
+            Application.Exit();
+        }
+
+        //Course select code
```
Comment (Gintasp) [inline View/Individual/LecturerView.cs]: Trinam

File: Controller/Lecturer/ILecturerController.cs [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+{
+    public interface ILecturerController
+    {
+        LecturerView LecturerView { get; set; }
+
+        void LoadLecturerData();
+        void HandleLeaveReviewClick();
+        void HandlePreviousFormClick();
+
+        //Not implemented in the form
```
Comment (Gintasp) [inline Controller/Lecturer/ILecturerController.cs]: Trinam

**Pull request merged**